stores/sql.go

```diff
@@ -362,6 +296,11 @@ func (s *SQLStore) Close() error {
 		return err
 	}
 
+	err = s.cs.Close()
```

Comment:
Shouldn't this be closed before the database considering that it is using it?

Reply:
we are closing it before we close the wallet on line 304. I guess the fact we fetch both DBs before calling `Close threw you so I'll move that closer to the db close method